README.md

```diff
@@ -10,6 +10,55 @@ The library is versioning following the [semantic versioning specification](http
 
 ## Usage
 
+### New build procedure
+A default node pipeline is now available and should be used in preference to the individual method calls in the `DefraUtils` class. This should very significantly reduce the amount of code required in your repository's Jenkinsfile:
+
+```
+@Library('defra-library@4') _
+
+buildNodeJs environment: 'dev'
```

Comment:
I think there should be a section on explaining exactly what steps `buildNodeJs` does as now it's a bit of a black box and also what parameters can be passed to it (currently only `environment`).

In order to use this build pipeline there are some conventions that teams are going to have to follow in their code so we should explain what they are.

Here are the ones I can think of.

- Helm chart must have the same name as the repository
- service name in Docker Compose must have the same name as the repository
- application must be deployed to `/home/node/` in container
- repository must be hosted on GitHub
- must supply FFC cluster as `environment` (only `dev` exists at present)
- must have separate deployment pipeline to trigger and we need a naming convention for name and token for this.  See my other comment as this is an issue we haven't addressed as part of this.